metanome-cli/README.md

```diff
@@ -0,0 +1,40 @@
+# Metanome CLI
+
+The main purpose of this project is to provide a command-line interface for [Metanome](https://github.com/HPI-Information-Systems/Metanome) to allow for easy automation of data profiling tasks, e.g., to conduct experiments or to profile datasets batchwise.
+Besides that, this project integrates Metanome with
+* [Metacrate](https://github.com/stratosphere/metadata-ms), a storage and analytics tool for data profiles, and
+* [ProfileDB](https://github.com/sekruse/profiledb-java), a tiny tool to collect and store experimental data.
+
+Furthermore, HDFS is supported as input source.
```

Comment:
This support could be migrated to the main project because it does not really fit into this subproject.